diagnostic.md

```diff
@@ -5,74 +5,96 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The backend allows you to save data and access that data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The model.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+The controller.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Our version of a view is the JSON that we will be passing to the front end. We
+don''t use the views provided because there are front ends which will display
+what we want better than the provided views.
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+C = create
+R = read
+U = update
+D = destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+The controller.
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```bash
-// your response here
+get => index => show all
+get => show => show one that is specified
+post => create => make something new
+patch => update => change this thing that i''ve specified, as i''ve specified
+delete => destroy => delete this thing that i''ve specified
+
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```bash
-// your response here
+*goes to the routes
+*goes to controller
+*goes to controller action
+*goes to model
+*goes to database
+*goes to model
+*goes to controller
+*goes back to client
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails new rails-api (assuming that the name of the app that you want to create is
+rails-api)
+
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+bundle exec rails server
 ```
 
 What are the commands to drop, create and migrate a database from the command
 line? (3 bullet points)
 
 ```bash
-// your response here
+* bundle exec rails create (name of the database)
+* bundle exec rake db:migrate
+* bundle exec rails destroy (name of the database)
+
```

Comment:
Close! I think you confused some rails and rake commands here, but I'm happy you remembered the importance the `bundle exec`.